app/js/SafeExec.js

```diff
@@ -87,3 +85,12 @@ function safeExecPromise(command, options) {
     })
   })
 }
+
+module.exports.runOnce = runOnce
```

Comment:
Does this need to be exported? I don't think it is used outside of this module.

Reply:
I wrote a quick unit-tests for it and this test needs access to this internal fn.

https://github.com/overleaf/filestore/pull/96/files#diff-ecdb1e7e03113c4eee7b501b26f99999

I can move it to a private `_runOnce` export if you would like.

Reply:
No, that makes sense - leave it as-is.